TLM/TLM/UI/LegacySubTool.cs

```diff
@@ -183,5 +203,20 @@ public abstract class LegacySubTool {
                 GUI.DragWindow();
             }
         }
+
+        public void OnUpdate(ModUI.UIOpacityNotification subject) {
+            Texture2D windowTexture = windowTexture_;
```

Comment:
Can you check if just
UnityEngine.Object.Destroy(windowTexture_);
windowTexture_ = null;
would do too?

Reply:
Game engine will throw error if you try destroying already destroyed object, that's why is that ugly